docs/zilevm/onboard/onboard-metamask.md

```diff
@@ -25,55 +25,59 @@ description: Wallets that can interact with ZILEVM
 
 !!! tip
 
-    You can send funds between ZILEVM and Zilliqa networks by translating
-    your Zilliqa address into it's base 16 representation which ZILEVM and
+    You can send funds between EVM and Zilliqa format APIs translating
+    your Zilliqa address into it's base 16 representation which
     Metamask can understand.
 
 ### Configure MetaMask with ZILEVM network
 
 To add a new network to MetaMask - click the current network selected at the top
 of the extenstion and press 'Add Network'.
 
```

Comment:
We might be able to streamline this now we're registered with the normal chainlists. I'll have a look...

Reply:
Will update in a subsequent PR if so, not a blocker :)